meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname

```diff
@@ -5,7 +5,8 @@ set -e
 . /usr/sbin/resin-vars
 
 if [ -z "$CONFIG_HOSTNAME" ]; then
-    CONFIG_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH)
+    # take just the first 7 characters
+    CONFIG_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH | sed -e 's/\(.......\).*/\1/')
```

Comment:
Sorry, the curly braces should be escaped actually, so they would not be extended regex I guess